lib/std/terminfo.zig

```diff
@@ -0,0 +1,1122 @@
+    .{ .short = "u0", .long = "user0" },
+    .{ .short = "u1", .long = "user1" },
+    .{ .short = "u2", .long = "user2" },
+    .{ .short = "u3", .long = "user3" },
+    .{ .short = "u4", .long = "user4" },
+    .{ .short = "u5", .long = "user5" },
+    .{ .short = "u6", .long = "user6" },
+    .{ .short = "u7", .long = "user7" },
+    .{ .short = "u8", .long = "user8" },
+    .{ .short = "u9", .long = "user9" },
+    .{ .short = "wait", .long = "wait_tone" },
+    .{ .short = "xoffc", .long = "xoff_character" },
+    .{ .short = "xonc", .long = "xon_character" },
+    .{ .short = "zerom", .long = "zero_motion" },
+    .{ .short = "scesa", .long = "alt_scancode_esc" },
+    .{ .short = "bicr", .long = "bit_image_carriage_return" },
+    .{ .short = "binel", .long = "bit_image_newline" },
+    .{ .short = "birep", .long = "bit_image_repeat" },
+    .{ .short = "csnm", .long = "char_set_names" },
+    .{ .short = "csin", .long = "code_set_init" },
+    .{ .short = "colornm", .long = "color_names" },
+    .{ .short = "defbi", .long = "define_bit_image_region" },
+    .{ .short = "devt", .long = "device_type" },
+    .{ .short = "dispc", .long = "display_pc_char" },
+    .{ .short = "endbi", .long = "end_bit_image_region" },
+    .{ .short = "smpch", .long = "enter_pc_charset_mode" },
+    .{ .short = "smsc", .long = "enter_scancode_mode" },
+    .{ .short = "rmpch", .long = "exit_pc_charset_mode" },
+    .{ .short = "rmsc", .long = "exit_scancode_mode" },
+    .{ .short = "getm", .long = "get_mouse" },
+    .{ .short = "kmous", .long = "key_mouse" },
+    .{ .short = "minfo", .long = "mouse_info" },
+    .{ .short = "pctrm", .long = "pc_term_options" },
+    .{ .short = "pfxl", .long = "pkey_plab" },
+    .{ .short = "reqmp", .long = "req_mouse_pos" },
+    .{ .short = "scesc", .long = "scancode_escape" },
+    .{ .short = "s0ds", .long = "set0_des_seq" },
+    .{ .short = "s1ds", .long = "set1_des_seq" },
+    .{ .short = "s2ds", .long = "set2_des_seq" },
+    .{ .short = "s3ds", .long = "set3_des_seq" },
+    .{ .short = "setab", .long = "set_a_background" },
+    .{ .short = "setaf", .long = "set_a_foreground" },
+    .{ .short = "setcolor", .long = "set_color_band" },
+    .{ .short = "smglr", .long = "set_lr_margin" },
+    .{ .short = "slines", .long = "set_page_length" },
+    .{ .short = "smgtb", .long = "set_tb_margin" },
+    .{ .short = "ehhlm", .long = "enter_horizontal_hl_mode" },
+    .{ .short = "elhlm", .long = "enter_left_hl_mode" },
+    .{ .short = "elohlm", .long = "enter_low_hl_mode" },
+    .{ .short = "erhlm", .long = "enter_right_hl_mode" },
+    .{ .short = "ethlm", .long = "enter_top_hl_mode" },
+    .{ .short = "evhlm", .long = "enter_vertical_hl_mode" },
+    .{ .short = "sgr1", .long = "set_a_attributes" },
+    .{ .short = "slength", .long = "set_pglen_inch" },
+};
+
+pub fn getCapsLookUpTable() !std.StringHashMap([]const u8) {
+    var data = std.StringHashMap([]const u8).init(std.heap.page_allocator);
```

Comment:
Never assume an allocator in the standard library. Have the user pass an `allocator`.